R/reportEDGETransport.R

```diff
@@ -284,89 +284,37 @@ reportEDGETransport <- function(output_folder=".",
 
 
   ## Demand emissions
-  reportingEmi <- function(repFE, gdx, miffile){
+  reportingEmi <- function(repFE, gdx){
 
     ## load emission factors for fossil fuels
-    p_ef_dem <- readGDX(gdx, "p_ef_dem")  ## MtCO2/EJ
-    p_ef_dem <- as.data.table(p_ef_dem)[all_enty %in% c("fepet", "fegas", "feelt", "feh2t")]  ## emissions factor for Electricity and Hydrogen are 0, as we are calculating tailpipe emissions
+    p_ef_dem <- readgdx(gdx, "p_ef_dem")[all_enty %in% c("fepet", "fedie", "fegas")]  ## MtCO2/EJ
+    p_ef_dem[all_enty == "fegas", all_enty := "fegat"]
```

Comment:
is it because there is not emi factor for fegat?

Reply:
yes, it looks just like it. Anyways, the difference should not really matter (be smaller than fedie vs fepet).